src/api-object.ts

```diff
@@ -136,6 +136,12 @@ export class ApiObject extends Construct {
         ...props.metadata?.labels,
       },
     });
+
+    const constrctMetadata = this.chart.node.tryFindChild('ConstructMetadata') as ApiObject;
+    if (constrctMetadata) {
+      constrctMetadata.addJsonPatch(JsonPatch.add(`/data/${this.name}.path`, this.node.path ));
+    }
```

Comment:
NIT: `constructMetadata` spelling 

```suggestion
    const constructMetadata = this.chart.node.tryFindChild('ConstructMetadata') as ApiObject;
    if (constructMetadata) {
      constructMetadata.addJsonPatch(JsonPatch.add(`/data/${this.name}.path`, this.node.path ));
    }
```